utils/notifications.go

```diff
@@ -39,7 +32,8 @@ func ConstructNotificationPayload(userIDs []string, heading, content string) Not
 }
 
 func SendNotification(payload NotificationPayload) error {
-	apiKey := GetAPIKey()
```

Comment:
Make this a method on the App struct later and use the config instead so we don't keep retrieving the config stuff again and again.